packages/models-library/src/models_library/api_schemas_dynamic_sidecar/ports.py

```diff
@@ -0,0 +1,35 @@
+from enum import auto
+
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from models_library.services_types import ServicePortKey
+from models_library.utils.enums import StrAutoEnum
+from pydantic import BaseModel
+
+
+class OutputStatus(StrAutoEnum):
```

Comment:
I am a bit concerned that we have many of these states and status all over the place... duplication is evil, and makes everything more complex. be careful please.